app/main.py

```diff
@@ -20,10 +27,114 @@
 app.include_router(email.router)
 
 
+def add_countries_to_db():
+    """
+    Adding all new countries to the "Country" table in the database.
+    Information is based on the "countries" list.
+    (The list is located in app/database/countries.py)
+    Names are described either as:
+    "Country Name, City Name" or
+    "Country Name" solely.
+    Timezones are described as "Continent/ City Name"
+    for example:
+        name: Israel, Jerusalem
+        timezone: Asia/Jerusalem
+    """
+    session = SessionLocal()
+    for country in countries:
+        partial_name = country['name']
+        for capital in country['timezones']:
+            capital_name = capital.split('/')[-1]
+            if partial_name != capital_name:
+                name = partial_name + ', ' + capital_name
+            else:
+                name = capital_name
+            existing = session.query(Country).filter_by(name=name).first()
```

Comment:
This might take long time to execute due to the large amount of queries we perform here.

Prefer using `on conflict ignore` to simplify this part of the code, and to get better performance.